otterdog/adoptium.jsonnet

```diff
@@ -1,5 +1,29 @@
 local orgs = import 'vendor/otterdog-defaults/otterdog-defaults.libsonnet';
 
+local jdkName(name) =
+  if std.startsWith(name, "jdk") then 
+    "JDK%s" % [ std.substr(name, 3, std.length(name) - 3)]
+  else
+    name;
+
+local newMirrorRepo(repoName) = orgs.newRepo(repoName) {
+  jdk_name:: jdkName(repoName),
```

Comment:
Probably OK, but IMHO no need to be so fancy and parse the repo name - which is not enforced but just a standard. I would have just set the default `jdk_name` to be "OpenJDK" and default `jdk_url` to be "https://github.com/openjdk".

No objection to this version though.

Reply:
I tried to adhere to the existing descriptions which use a phrase like JDKxx mirror. Changing that to OpenJDK should make the configuration simpler, if you are ok with that I can make the change.